NuGet/tools/install.ps1

```diff
@@ -0,0 +1,3 @@
+﻿param($installPath, $toolsPath, $package, $project)
+
+Register-GlimpseExtension $package $DTE
```

Comment:
It is not necessary, Glimpse.core includes the ps1 with the definition of that method. Other extensions only include this reference.